### PR TITLE
clDNN backend optimization

### PR DESCRIPTION
**Pull request by huningxin**
fix https://github.com/intel/webml-polyfill/issues/178

### CODE REVIEW

**Comment by huningxin** [comment]
@fujunwei , please take a look. Thanks!

File: services/ml/model_impl_win.cc [hunk 1]
```diff
@@ -599,6 +599,10 @@ int32_t ModelImplWin::CldnnAddElementwise(
     return mojom::BAD_DATA;
   }
 
+  // Setup output quanitization factors
+  std::string empty("");
```
Comment (fujunwei) [inline services/ml/model_impl_win.cc]: The default [constructor ](http://www.cplusplus.com/reference/string/string/string/)of std::string will construct an empty string.
`std::string empty;`

File: services/ml/model_impl_win.cc [hunk 2]
```diff
@@ -405,6 +412,46 @@ int32_t ModelImplWin::CldnnAddInputLayout(uint32_t index) {
...
+      .output_data_type = cldnn_f32,
+  };
+  // Setup inputs.
+  std::vector<cldnn_primitive_id> input_ids_array(1);
+  input_ids_array[0] = output_id_str.c_str();
+  reorder_desc.input = {.data = input_ids_array.data(),
+                        .size = input_ids_array.size()};
+  // Setup mean mode.
+  const std::string empty("");
+  reorder_desc.mean_subtract = empty.c_str();
```
Comment (fujunwei) [inline services/ml/model_impl_win.cc]: Ditto.

**Comment by huningxin** [comment]
@fujunwei , thanks for review! I've addressed your comments with a new commit. Please take another look.

**Comment by fujunwei** [comment]
LGTM.

**Comment by huningxin** [comment]
Thanks! I am going to merge.

**Pull request merged**